ports/esp32/mbedtls/mbedtls_config.h

```diff
@@ -0,0 +1,7 @@
+// Time mbedtls_platform
+#include <time.h>
+#include <sys/time.h>
+
+#define MBEDTLS_HAVE_ASM
+
+time_t platform_mbedtls_time(time_t *timer);
```

Comment:
I see that this file is indeed included in the build.  But that seems to be by accident: `esp-idf/components/mbedtls/port/include/mbedtls/esp_config.h` includes `mbedtls/mbedtls_config.h` which is intended to pick up the default `mbedtls_config.h` from the `mbedtls` component of the IDF.

So having this file here will completely override the default set of settings.

Why is this file needed?  I don't think the `MBEDTLS_HAVE_ASM` option does anything, that seems to only be needed for x84 architectures.